### ISSUE TITLE
Some coordinate based waypoints seem to be missing

### ISSUE THREAD

**Issue by JRascagneres** [flybywiresim/a32nx#6341]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-12-09T22:17:20+00:00",
  "ref": "master",
  "sha": "6e17701190a2818c76db1df975ffe6492d641fa4",
  "actor": "Benjozork",
  "event_name": "manual"
}
```

### Describe the bug

```
PHNL/04R MKK4 APACK DCT ZOULU DCT 30N157W 35N157W 40N157W 45N156W 50N155W 55N153W DCT ODK DCT BAITT PANC/33
```

My following flight plan generated by simbrief / navigraph cannot be fully imported into the A32NX. Most of the 'coordinate' waypoints don't work. `50N155W` and `55N153W` seem to import no problem but the other 'coordinate' ones don't work: `30N157W`, `35N157W`, `40N157W` and `45N156W`.

Unsure what the reason is, especially as some work and some don't. I have navigraph fully updated incase these are treated as regular waypoints.

### Expected behavior

I expect the waypoints to be accepted by the MCDU.

### Steps to reproduce

Try to input any of `30N157W`, `35N157W`, `40N157W` and `45N156W` into the MCDU as a waypoint.

### Discord Username (optional)

Scorp#3465

**Comment by tshomas** [comment]
Just for confirmation, have seen this in support channel from multiple users. 

Some coordinates work fine and others do not, I usually get the users to use this image below that should show the correct conversions:
![0AA62686-D078-4C26-B81E-060A4911746E](https://user-images.githubusercontent.com/74841560/145620918-53e51ab5-5bd1-4cb2-a3f2-330b04a42674.png)

The last user with this issue I saw also had a problem with waypoints in the far W of longitude, the same as the OP, no idea if it’s just a certain area or waypoints that are at the far extreme of the coordinate system.

**Comment by tracernz** [comment]
Fixed by #6144
